apps/src/lab2/responseValidators.ts

```diff
@@ -52,3 +80,32 @@ export class ValidationError extends Error {
     Object.setPrototypeOf(this, ValidationError.prototype);
   }
 }
+
+export function setValidatorForAppType(appName: AppName) {
+  if (appName === 'pythonlab') {
+    Lab2Registry.getInstance().setSourceResponseValidator(
```

Comment:
If you store the result of `Lab2Registry.getInstance()` in a variable on line 85, can you simplify the subsequent code? i.e.,

```
  const registry = Lab2Registry.getInstance();
  if (appName === 'pythonlab') {
    registry.setSourceResponseValidator(PythonSourceResponseValidator);
  } else if (BLOCKLY_LABS.includes(appName)) {
    registry.setSourceResponseValidator(BlocklySourceResponseValidator);
  } else {
    registry.setSourceResponseValidator(DefaultSourceResponseValidator);
  }
  ```

Reply:
good call, will update